test/data_map_test.exs

```diff
@@ -0,0 +1,162 @@
+defmodule DataMapTest do
+  use PowerAssert
+
+  test "Insert empty row must fail" do
+    c1 = %Masdb.Schema.Column{is_pk: true,  name: "c1", type: :int}
+    c2 = %Masdb.Schema.Column{is_pk: false, name: "c2", type: :int}
+    c3 = %Masdb.Schema.Column{is_pk: false, name: "c3", type: :int}
+
+    timestamp = Masdb.Timestamp.get_timestamp()
+    inserted_value = %{}
+    inserted_schema = %Masdb.Schema{name: "foo", replication_factor: 0, columns: [c1, c2, c3]}
+    inserted_nodeid = "foo@127.0.0.1"
+
+    map = %Masdb.Data.Map{
+      node_id: inserted_nodeid,
+      last_update_time: timestamp,
+      last_sync_time: timestamp,
+      map: %{"foo" => %Masdb.Data.Table{}}
+    }
+
+    {_, {:cannot_insert_empty_row, _}} = Masdb.Data.Map.insert(map, inserted_schema, inserted_value)
```

Comment:
Please use asserts in all the tests. If we break something, having a nice assert will be awesome for the error logs.
```elixir
{_, {error, _}} = Masdb.Data.Map.insert(map, inserted_schema, inserted_value)
assert error == :cannot_insert_empty_row
```